### ISSUE TITLE
Enhancement: display full day name

### ISSUE THREAD

**Issue by hishambakr** [TouchBoarder/weekdays-buttons-bar#3]
In English, it is easy to recognize that SUN mains Sunday. But in Arabic, you need to display full name. this is not possible with weekdays-buttons-bar as it doesn't display full name. Even if it did that, it doesn't wrap to new line or scroll horizontally

**Comment by TouchBoarder** [comment]
Have you tried to change the width of the item in the theme -> see Global Theming, or with the setViewWidth(min width),
with setNumbersOfLetters(min letters)
And change the font size if needed?

**Comment by TouchBoarder** [comment]
It's working for me? when I set the view width to 120 and number of letters to 10 the full*\* weekday name is displayed and the days gets scrollable on small screens? 

`weekdaysDataSource = new WeekdaysDataSource(this, R.id.weekdays)
                .setDrawableType(WeekdaysDrawableProvider.MW_ROUND_RECT)
                .setViewWidth(120)
                .setNumberOfLetters(10)
                .start(this);`

*\* But looks like for some reason I only get the 6 first letters of the name for my Locale? 
The source used for the weekday names is:
`DateFormatSymbols(getLocale()).getWeekdays()`

From the documentation:

> Returns the array of strings containing the full names of the days of the week.

Bug: I've found the bug causing this, and will update with a fix later.
`if (mNumberOfLetters >= label.length())
            mNumberOfLetters = label.length();
...
label.substring(0, mNumberOfLetters)
`
Fix:
`int subStringLength =  label.length();
        if (subStringLength > mNumberOfLetters)
            subStringLength = mNumberOfLetters;
...
label.substring(0, subStringLength)
`
